vars/pod.groovy

```diff
@@ -10,6 +10,8 @@
 //   serviceAccount: string
 //   secrets: []string secret name and/or my-secret:my-key:mount-path
 //   configMaps: []string
+//   name : string
+//   arches : string
```

Comment:
the caller should set the name by just handing this function the string to use. We shouldn't be passing this information in here.